<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bottomDialog</class>
 <widget class="QDialog" name="bottomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bottom dialog</string>
  </property>
  <widget class="QPushButton" name="pauseButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>pause/go</string>
   </property>
  </widget>
  <widget class="QSlider" name="sliderTime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>551</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>46</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>time</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
